packages/integration-tests/api/api.js

```diff
@@ -48,11 +57,32 @@ async function callCumulusApi({ prefix, functionName, payload: userPayload }) {
   finally {
     // Delete the user created for this request
     await userModel.delete(userName);
+    await accessTokenModel.delete({ accessToken });
   }
 
   return JSON.parse(apiOutput.Payload);
 }
 
+/**
+ * Check API Lambda response for errors.
+ *
+ * Invoking Lambda directly will return 200 as long as the Lambda execution
+ * itself did not fail, ignoring any HTTP response codes internal to the
+ * object returned by the execution. Manually check those codes so we can
+ * throw any encountered errors.
+ *
+ * @param {Object} response - the parsed payload of the API response
+ * @param {Array<number>} acceptedCodes - additional HTTP status codes to consider successful
+ * @throws {Error} - error from the API response
+ * @returns {Object} - the original response
+ */
+function verifyCumulusApiResponse(response, acceptedCodes = []) {
```

Comment:
After deploying the new API which expects JWTs, I had forgotten to update the integration test helpers for API requests, so they were all failing. However, those failures were not surfaced in the build logs, so I had to add debugging to trace them out.

The reason why no failures were being reported is that any successful Lambda execution is a 200 response for `callCumulusApi`. Whether the request itself failed based on our API logic is determined by the internal response returned by the Lambda. API gateway checks/expects the internal response from the Lambdas, but our integration test helpers do not because they invoke Lamda correctly.

This function introspects the response from invoking API lambdas and throws the error if the status code is not successful, which then allows any errors from API lambdas to appear in integration test logs